lib/diagnostic.rb

```diff
@@ -78,12 +94,12 @@ def initialize
 
 ## What will be the output from calling `HouseCat.new.say_hello`?
 # replace nil with your response
-Response.housecat_noise = nil
+Response.housecat_noise = "I am a HouseCat and I go 'meow'"
 
 ## Explain why this would be the output, based on the method lookup chain.
 # your response as a comment here
 ##
-
+Because @sound was overwritten by the new class House_cat.
 ##
```

Comment:
We are not really overwriting @sound because when we instantiate a new instance of HouseCat,
sound is defined. 
The question is more so asking how we have access to `say_hello` even though it's not defined on HouseCat